src/web/mjs/connectors/ScanManga.mjs

```diff
@@ -23,23 +21,8 @@ export default class ScanManga extends Connector {
     }
 
     async _getMangas() {
-        let request = new Request(new URL('/scanlation/scan.data.json', this.url), this.requestOptions);
```

Comment:
No need to remove, just set the headers that the website wants, so it will work again...
```javascript
request.headers.set('x-cookie', `_ga=GA1.2.137581646.${parseInt(Date.now()/1000)}; addtl_consent=1~${new Array(647).fill().map(() => Math.floor(41 * Math.random())).join('.')}`);
request.headers.set('x-referer', this.url + '/scanlation/liste_series.html');
request.headers.set('x-requested-with', 'XMLHttpRequest');
```

Reply:
i mean the site itself doesn't know what the website wants
the search  function doesn't work at all

the method used also gets a shit ton of manga and manhwa that aren't viewable (not on site anymore)
i believe it moved to novels only

Reply:
i legit have no idea where u got this from